docs/guide/Titanium_SDK/Titanium_SDK_FAQ.md

```diff
@@ -5,8 +5,60 @@ weight: '50'
 
 # Titanium SDK FAQ
 
+<div class="faq">
+
 This document provides answers and links to commonly asked questions about Titanium SDK.
 
+## How to install Titanium Mobile
```

Comment:
Should we use the `Titanium SDK` naming here?

Reply:
I think we should do that in a separate PR. There are many more places since we've used the other name before